static/src/inline-svgs/journalism/audio-flagship/flagship-audio-series-header.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="538" height="214" viewBox="0 0 538 214">
```

Comment:
Let's optimise those SVGs in a follow-up PR.

Reply:
sure thing